### ISSUE TITLE
add moving rigid body testcases in fluids and including collision tests

### ISSUE THREAD

**Issue by prudhomm** [feelpp/feelpp#1837]
minimal testcases should be built to verify the proper behavior of the motion of the rigid bodies as well as the case of collision

**Comment by prudhomm** [comment]
Do we have a test for collision in sequential and parallel?
It should not take too much comuting time and we should have a metric to test if computation is ok
@lberti @VLCeline

**Comment by VLCeline** [comment]
@prudhomm We do not yet have a test with a metric that verifies if computations are correct. I will work on it next week.

**Comment by prudhomm** [comment]
@VLCeline @lberti where do we stand on this ?
We will need to keep a test or two to check fondamental properties (collision...) and move the other one in `feelpp/swimmer`

**Comment by VLCeline** [comment]
@prudhomm I have implemented three quick test cases in 2D:

1. Disk - wall interaction
<img width="293" alt="sphere-wall" src="https://user-images.githubusercontent.com/77676706/177169758-4af625da-3fcf-4618-9cc5-f9d05330d7ab.png">
During the iterations, I verify that the vertical position of the mass center decreases and remains inferior to r $+ \rho $, where r the disk radius and $\rho$ the width of the lubrication zone. At the end of the simulation I check that the vertical translation velocity is close to $0$. The execution time of this test is about $123$ s ($100$ iterations).


2. Square - Square interaction 
<img width="293" alt="square-square" src="https://user-images.githubusercontent.com/77676706/177171288-6ae7324a-712f-4b7a-9ac0-f00579d02ca0.png">
During the iterations I control the vertical position of both squares. At final time, I verify that the vertical translation velocity of the two bodies is close to $0$. The execution time is $280$ s ($100$ iterations). 


3. Ellipse - wall interaction 
<img width="293" alt="ellipse-wall" src="https://user-images.githubusercontent.com/77676706/177171312-15834d74-9842-4b9b-a32e-c4d16a23c1db.png">

I control the position during the simulation and check the vertical translation velocity and rotation at final time. 
The execution time of this test is about $263$ s ($200$ iterations).

I also implemented a test for the three-sphere swimmer - wall interaction: I control the position of the articulated bodies and the translation velocity. But the execution time of this test is around $1090$ s ($200$ iterations).
